### DISCUSSION TITLE
Rosetta compatible with DJI RC Pro controller?

### DISCUSSION THREAD

**Discussion by estevezb** [RosettaDrone/rosettadrone#180] [General]
Anyone know whether Rosetta is compatible with the DJI RC Pro controller?

**Comment by adpmdrones** [comment]
We've compiled the app with DJI SDK v5 but we got a google play services error.
Anyone have  the Google Play Services APK for DJI RC Pro controller?
![main_screen](https://github.com/RosettaDrone/rosettadrone/assets/4736951/c8baadab-614d-45b5-8a4e-4a51c0e88203)
